charts/ingress-nginx/Chart.yaml

```
@@ -1,6 +1,6 @@
 apiVersion: v1
 name: ingress-nginx
-version: 2.4.0
+version: 2.4.1
```

```suggestion
version: 2.5.0
```

As we are adding a new feature lets bump the minor version of the chart.

---

You lost this change